src/main/kotlin/net/lumalyte/lg/infrastructure/services/ARMIntegrationService.kt

```
@@ -5,81 +5,72 @@ import org.bukkit.Location
     }
 
     /**
-     * Check if ARM integration is enabled
+     * Check if EnthusiaMarket integration is available
      */
-    fun isEnabled(): Boolean = armEnabled
+    fun isEnabled(): Boolean {
+        return try {
+            Class.forName("net.badgersmc.em.api.ShopGuildLookup")
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)